spec/requests/authentication_spec.rb

```diff
@@ -237,6 +237,17 @@
           expect(response.parsed_body["token_ttl"]).to eq(::Settings.session.timeout.to_i_with_method)
         end
 
+        it "gets a token based identifier with a updated UI based token_ttl" do
+          ::Settings.session.timeout = 300
+          api_basic_authorize
+
+          get api_auth_url, :params => { :requester_type => "ui" }
+
+          expect(response).to have_http_status(:ok)
+          expect_result_to_have_keys(%w(auth_token token_ttl expires_on))
+          expect(response.parsed_body["token_ttl"]).to eq(300)
```

Comment:
Not sure if above tests the change in flight of Settings.session.timeout, we might need 2 get's like we do in some other tests.